sqlalchemy_utils-stubs/primitives/weekday.pyi

```diff
@@ -1,5 +1,21 @@
 from .. import i18n as i18n
 from ..utils import str_coercible as str_coercible
-from typing import Any
 
-def __getattr__(name: str) -> Any: ...  # incomplete
+
+@str_coercible
+class WeekDay:
+    NUM_WEEK_DAYS: int
+
+    def __init__(self, index: int) -> None:
+        pass
+
+    def get_name(self, width:str='wide', context:str='format') -> str:
+        pass
```

Comment:
```suggestion
    def get_name(self, width: str = ..., context: str = ...) -> str:
        pass
```

Using `...` for default values [seems to be the recommended way for stubs](https://mypy.readthedocs.io/en/stable/stubs.html#stub-file-syntax). (Also fixes the spacing a bit)